src/main/java/org/openelisglobal/common/provider/query/TestResultLimitsProvider.java

```diff
@@ -110,8 +110,11 @@ public int compare(ResultLimit e1, ResultLimit e2) {
                 resultLimitObject.put("highValid", resultLimit.getHighValid());
                 resultLimitObject.put("highReportingRange", resultLimit.getHighReportingRange());
                 resultLimitObject.put("lowReportingRange", resultLimit.getLowReportingRange());
-                resultLimitObject.put("highCriticalRange", resultLimit.getHighCriticalRange());
-                resultLimitObject.put("lowCriticalRange", resultLimit.getLowCriticalRange());
+                resultLimitObject.put("highRangeHigh", resultLimit.getHighRangeHigh());
```

Comment:
I’d like variable names to continue to mention critical ranges, so the concept is strongly established in the code across all classes.

using case sensitive find and replace:

highRangeHigh -> highCriticalRangeHigh
highRangeLow -> highCriticalRangeLow
lowRangeHigh -> lowCriticalRangeHigh
lowRangeLow -> lowCriticalRangeLow

HighRangeHigh -> HighCriticalRangeHigh
HighRangeLow -> HighCriticalRangeLow
LowRangeHigh -> LowCriticalRangeHigh
LowRangeLow -> LowCriticalRangeLow